gqlgen.yml

```diff
@@ -92,6 +92,10 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.APIExternalLink
   ExternalLinkInput:
     model: github.com/evergreen-ci/evergreen/rest/model.APIExternalLink
+  ServiceFlags:
+    model: github.com/evergreen-ci/evergreen/rest/model.APIServiceFlags  
+  ServiceFlagsInput:
+    model: github.com/evergreen-ci/evergreen/rest/model.APIServiceFlags  
```

Comment:
nit. Could we put these in alphabetical order (in reference to the other models in this file)?

Reply:
crying i thought i put them in their alphabetical order placement